docker-compose.yml

```diff
@@ -218,11 +218,15 @@ services:
       - "traefik.docker.network=learning-env_reverse-proxy"
       - "traefik.enable=true"
   shibboleth-idp-ui:
-    build:
-      context: ./shibboleth-idp-ui
+    image: unicon/shibui:latest
     command:
+      - "/usr/bin/java"
```

Comment:
do we need these? this should be the entrypoint of our image

Reply:
Fixed! Also fixed a leftover merge conflict issue.